draft-tigress-requirements.md

```diff
@@ -186,18 +186,13 @@ A number of existing solutions / protocols have been reviewed in order to be use
 
 ## Arbitrary Messaging Channel (Email / WhatsApp / SMS / Signal / etc.)
 
-The Provisioning Information MAY be sent from Sender to Receiver over an arbitrary messaging channel that supports binary file transfer, but this would not support provisioning flows which require multiple round trips as requied by (Req-RoundTrips).
+The Provisioning Information MAY be sent from Sender to Receiver over an arbitrary messaging channel that supports binary file transfer, but this would not support provisioning flows which require multiple round trips as requied by (Req-RoundTrips). The same requirement applies to Signal protocol outside of the Signal app, as the Req-RoundTrips would likely be difficult and add a lot of friction for the user. 
```

Comment:
I think reference Signal is not appropriate here, the requirement here applies to messaging application, the paragraph about Signal - to a messaging protocol. Signal protocol may be (and probably have been) implemented in applications other than Signal Messaging Application

Reply:
What could be added here though, is a reference to the requirement of separation of channels - but we decided to remove this requirement...